### ISSUE TITLE
[Build] [0010] Objects under build seen by server

### ISSUE THREAD

**Issue by mwpowellhtx** [mwpowellhtx/KP-Liberation#60]
### Basic Information
**Mission version:** `0.98.dev-s10`
**Map used:** _Altis_

### Short Issue Description
Apparently objects under build can be "seen" by the server. Scenario in question, running logistics line between FOB and factory sector. FOB has no storage, player builds storage. The storage is "seen" by the UNLOADING state. Crates momentarily "attach" to the object being built. Once build is complete, they are no longer attached to anything.

We are kind of on the fence over this one. Should at least verify whether the storage object has "seen" a UUID or marker name yet in order to include it among the eligible storage objects.

That being stated as a plausible goal, is it that terrible that crates do not attach after all?

**Comment by mwpowellhtx** [comment]
TBD: Specifically, `KPLIB_fob_originalUuid` and `KPLIB_sector_markerName` variables should be set in order for storage to be considered eligible for logistics, or other, activities. The queries for FOB and factory storage objects does take these elements into account, so the question then becomes is, truly "who" sees the object being built prior to acknowledgment? Or perhaps such objects need to be further "proxied" with flags stating as much.

**Comment by mwpowellhtx** [comment]
Closing this one. This is pretty much the approach I ended up taking, add a UUID to the objects. But more importantly is simply to understand that objects under build mode itself are created local only, until such time that the builder acknowledges them, then their details are posted to the server and an object is recreated, effectively. May open up another issue, because actually have some other ideas how that could be better handled transitioning from in builder FOB zone view to live and available server wide.
